landing-pages/site/content/en/community/_index.html

```diff
@@ -16,10 +16,10 @@
             </p>
             <ul class="ticks-blue mx-auto">
                 <li>
-                    Users list: <a class="list-link" href="mailto:users-subscribe@airflow.apache.org">users-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:users-unsubscribe@airflow.apache.org">users-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
+                    Users list - for discussion, news, and information such as release details: <a class="list-link" href="mailto:users-subscribe@airflow.apache.org">users-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:users-unsubscribe@airflow.apache.org">users-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
```

Comment:
In fact I think most of the discussion and information sharing takes place on `dev`. If I'm not mistaken the `users` list is for users seeking answers to how-to questions / troubleshooting. The dev list is mostly for project related discussion - new features discussion, release voting, planning etc. So it's not only for contributors.